netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
@jainvipin 

Thanks for your elaborate reply. From your response, I think we are on same page wrt what this PR doesn't take cares of and issues that will still need to be tracked/addressed. And I am fine leaving them open for now.

Last question, what would we miss if for 0.1 release we were to let user issue `netmaster --version` command on the host where netmaster is running? Is it just the ease of issuing the command from any host?